content/en/tracing/advanced/opentracing.md

```diff
@@ -239,11 +238,39 @@ public class MyHttpRequestExtractAdapter implements TextMap {
   }
 }
 ```
+
 Notice the above examples only use the OpenTracing classes. Check the [OpenTracing API][1] for more details and information.
 
+#### Set errors
+
+To customize an error associated to one of your span, you must import the `io.opentracing.Span`, `io.opentracing.tag.Tags`, and `io.opentracing.util.GlobalTracer` library in the method where the error occurs:
+
+```java
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+```
+
+Then you need to set the error to `true` and add *at least* the `error.msg`, `error.type`, and `error.stack` tag to your span.
+
+```java
+    final Span span = GlobalTracer.get().activeSpan();
+    if (span != null) {
+      Tags.ERROR.set(span, true);
+      span.setTag("error.msg", "<ERROR_MESSAGE>");
+      span.setTag("error.type", "<EXCEPTION_NAME>");
+      span.setTag("error.stack", "<ERROR_STACK>");
```

Comment:
Replace these with
```java
span.log(Collections.singletonMap(Fields.ERROR_OBJECT, ex));
```

to better align with OpenTracing: https://github.com/opentracing/opentracing-java